core/src/main/java/org/infinispan/topology/ClusterCacheStatus.java

```diff
@@ -692,6 +694,17 @@ public void setRebalanceEnabled(boolean enabled) {
       }
    }
 
+   /**
+    * Only performs rebalance if the per-cache rebalancing is enabled
+    */
+   public void conditionalRebalance() {
```

Comment:
`startQueuedRebalance` already checks if rebalancing is enabled, do we really need this method?